aksk/cmd/main.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2021 The SODA Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"os"
+
+	"github.com/micro/go-micro/v2"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/opensds/multi-cloud/aksk/pkg/iam"
+	handler "github.com/opensds/multi-cloud/aksk/pkg/service"
+	"github.com/opensds/multi-cloud/aksk/pkg/utils/config"
+	pb "github.com/opensds/multi-cloud/aksk/proto"
+	"github.com/opensds/multi-cloud/api/pkg/utils/obs"
+)
+
+func main() {
+	iamHost := os.Getenv("IAM_HOST")
+	iam.Init(&config.CredentialStore{
```

Comment:
This is very specific to Keystone. Why can't we read it from config?

Reply:
That is right it specific, having Configurable Cred mgmt , when the rest of the embedded code is Keystone specific doesn't make a lot of sense. However, configurable IAM is the way forward , and it will be addressed when the IAM is an independent service . For now , this should suffice.